scenarios/test_codegen/test/ChainManager_test.res

```diff
@@ -357,13 +348,13 @@ describe("determineNextEvent", () => {
             },
         )
 
-        let {val: {earliestEvent}} = determineNextEvent_unordered(fetchStatesMap)->Result.getExn
+        // let {val: {earliestEvent}} = determineNextEvent_unordered(fetchStatesMap)->Result.getExn
 
-        Assert.deepEqual(
-          earliestEvent->FetchState_test.getItem,
-          Some(singleItem),
-          ~message="Should have taken the single item",
-        )
+        // Assert.deepEqual(
+        //   earliestEvent->FetchState_test.getItem,
+        //   Some(singleItem),
+        //   ~message="Should have taken the single item",
+        // )
```

Comment:
Do you want to delete these tests or is the hope to reimplement them? Rather not leave dead commented code